package.json

```diff
@@ -25,7 +25,8 @@
     "clean": "rm -r dist/",
     "test": "mocha --config .mocharc.json lib/**/*.test.ts",
     "lint": "eslint . --ext .ts",
-    "lint:fix": "npm run lint -- --fix"
+    "lint:fix": "npm run lint -- --fix",
+    "prepublish": "npm run clean; npm run build"
   },
   "dependencies": {
     "@auth0/fga": "^0.9.1",
```

Comment:
Ideally, we only want openfga/sdk, not auth0/fga

Reply:
This is not the official OpenFGA CLI, which allows us to make use of the SDK to support deploying to Auth0 FGA environments.

See: https://github.com/rhamzeh/openfga-cli/blob/main/lib/helpers/openfga/fga.adapter.ts#L33